config/numpy.sh

```diff
@@ -1,5 +1,5 @@
 # One could also set NPY_BLAS_LIBS, NPY_LAPACK_LIBS and NPY_CBLAS_LIBS since numpy 1.21.
-PRE_BUILD_COMMANDS='module load flexiblas/3.0.4;
+PRE_BUILD_COMMANDS="module load flexiblas/$FLEXIBLAS_VERSION;"'
```

Comment:
if it's ok to use the latest from the current year, then we could remove this variable `FLEXIBLAS_VERSION`

Reply:
done